chiboycalix/kaitechApp
# Setup redux and consume endpoint

#### What does this PR do?
> - Setup Redux for the application
> - Setup axios to make HTTP calls
#### Description of Task to be completed?
> - Install redux, redux-thunk, redux devtools, axios
> - Setup your components to be able to communicate with redux
#### How should this be manually tested?
> - Clone the repo
> - run `npm install` to install all the required dependencies
> - run `npm start` to spin up the development server. This will automatically open up your application in your default browser and you should be able to see the image below
<img width="1280" alt="Screen Shot 2019-11-06 at 2 46 45 PM" src="https://user-images.githubusercontent.com/35861980/68303510-4b870d80-00a4-11ea-9061-0670a9c0f36d.png">
